nano/node/json_handler.cpp

```diff
@@ -283,6 +283,40 @@ nano::amount nano::json_handler::amount_impl ()
 	return result;
 }
 
+nano::amount nano::json_handler::decimal_amount_impl ()
+{
+	nano::amount wholeNumber (0);
+	nano::amount decimalNumber (0);
+	nano::amount result (0);
+	if (!ec)
```

Comment:
Let's make this a guard if and save a whole level of if nesting for better readability.

```
if (ec) return result;
```